test/metric_value_benchmark/base_test.go

```diff
@@ -58,10 +60,17 @@ func (t *TestRunner) runAgent() (status.TestGroupResult, error) {
 		},
 	}
 
+	extraCommands := t.testRunner.getExtraCommands()
+	err := util.RunCommands(extraCommands)
+	if err != nil {
+		testGroupResult.TestResults[0].Status = status.FAILED
+		return testGroupResult, fmt.Errorf("Failed to run extra commands due to: %v", err.Error())
```

Comment:
nit - `err.Error()` is a string. You should either change it to `err` and leave `%v` or change it to `%s` and leave `err.Error()`

Reply:
Nice catch and thanks for reminding me. Fixed it !!!